content/reference-ledger-format.md

```diff
@@ -303,6 +305,72 @@ The `index` of an Offer node is the SHA-512Half of the following values put toge
 * The Sequence number of the transaction that created the offer
 
 
+
+## PayChannel ##
+[[Source]<br>](https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/impl/LedgerFormats.cpp#L134 "Source")
+
+_(Requires the [PayChan Amendment](concept-amendments.html#paychan).)_
+
+The `PayChannel` node type represents a payment channel, which is a structure that makes it easier to perform asynchronous, rapid payments and micropayments of XRP. A payment channel holds a balance of XRP that can only be paid out to a specific destination address until the channel is closed. Any unspent XRP is returned to the channel's owner (the source address that created and funded it) when the channel closes.
```

Comment:
I think that description from the code is still a little obtuse. Here's my attempt at a clearer wording:

> The `PayChannel` node type represents a payment channel. Payment channels enable small, rapid off-ledger payments of XRP that can be later reconciled with the consensus ledger.